src/middlewares/authentication.js

```diff
@@ -0,0 +1,74 @@
+import jwt from 'jsonwebtoken';
+import 'dotenv/config';
+
+/**
+  * @description class representing User Authentication with JWT
+  *
+  * @class VerifyUser
+  */
+class VerifyUser {
+  /**
+    * @description - This method is responsible for generating user token
+    *
+    * @static
+    * @param {object} payload - Object representing encoded data that makes up the token
+    * @param {object} time - Object represnting time taken for the token to expire
+    *
+    * @returns {object} - object representing response message
+    *
+    * @memberof UserHandler
```

Comment:
Awesome work, @marcdomain, flawless! Well, almost.

On this line (`19`) shouldn't the `generateToken` method be a member of the `VerifyUser` class as opposed to `UserHandler` that is currently depicted by the JSDoc comment?

Same applies to lines `38`